test/Microsoft.ML.Tests/TrainerEstimators/MatrixFactorizationTests.cs

```diff
@@ -96,10 +96,10 @@ public void MatrixFactorizationSimpleTrainAndPredict()
             // MF produce different matrices on different platforms, so check their content on Windows.
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                Assert.Equal(0.301269173622131, leftMatrix[0], 5);
-                Assert.Equal(0.558746933937073, leftMatrix[leftMatrix.Count - 1], 5);
-                Assert.Equal(0.27028301358223, rightMatrix[0], 5);
-                Assert.Equal(0.390790820121765, rightMatrix[rightMatrix.Count - 1], 5);
+                Assert.Equal(0.290507137775421, leftMatrix[0], 5);
+                Assert.Equal(0.558072924613953, leftMatrix[leftMatrix.Count - 1], 5);
```

Comment:
Do we know why these baselines are changing?

Reply:
Yes, because MF result is corresponding to nr_bins, we change how we initialize nr_bins, so the baseline also changed